components/core/config/config.ts

```diff
@@ -296,6 +296,7 @@ export interface TableConfig {
   nzShowSizeChanger?: boolean;
   nzSimple?: boolean;
   nzHideOnSinglePage?: boolean;
+  nzSortDirections?: Array<string | null>;
```

Comment:
what about migrate `NzTableSortOrder` to `ng-zorro-antd/core/types` ?

if use ` Array<string | null>`, code auto-completion and type check won't work

Reply:
I'm not sure about this, it would be a breaking change and it's not worth the bother.

Maybe we can simplify and replace all occurences of `NzTableSortOrder` with `string | null`?
Currently NzTableSortOrder is like that : `export type NzTableSortOrder = string | 'ascend' | 'descend' | null;`

Reply:
So, what should I do ?

Reply:
What about add a document link here
```suggestion
  /**
   * @see {@link NzTableSortOrder}
   */
  nzSortDirections?: Array<string | 'ascend' | 'descend' | null>;
```